src/editors/table.js

```diff
@@ -64,7 +64,12 @@ JSONEditor.defaults.editors.table = JSONEditor.defaults.editors.array.extend({
 
     this.panel.appendChild(this.table);
```

Comment:
A better solution would be to change the order of the of the table and panel.
````javascript
    this.controls = this.theme.getButtonHolder();   
    if (!this.table_row_top) {
      this.panel.appendChild(this.table);
      this.panel.appendChild(this.controls);
    }
    else {
      this.panel.appendChild(this.controls);
      this.panel.appendChild(this.table);
    }
````